java/vector/src/main/java/org/apache/arrow/vector/FixedSizeBinaryVector.java

```diff
@@ -116,6 +117,18 @@ public byte[] get(int index) {
     return dst;
   }
 
+  /**
+   * Read the value at the given position to the given output buffer.
+   * The caller is responsible for checking for nullity first.
+   *
+   * @param index position of element.
+   * @param outputBuffer the buffer to write into.
+   */
+  public void read(int index, ReusableBuffer<?> outputBuffer) {
```

Comment:
I hate to bike shed, but since these names would end up in docs, auto-complete and API browsing...

Is "output" superfluous here, since it's a void method?